project/plugins.sbt

```diff
@@ -0,0 +1,9 @@
+scalaVersion := "2.10.6"
+
+addSbtPlugin("com.eed3si9n" % "sbt-doge" % "0.1.5")
+addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.5.4")
+addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.8.5")
```

Comment:
Previously brought in transitively by sbt-ghpages. Added explicitly to use the latest release.